src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml

```diff
@@ -349,21 +449,46 @@ limitations under the License.
       <arg name="DaysToReturn" type="ScheduleDayOfWeekBitmap" min="0x0" max="0xFF"/>
       <arg name="ModeToReturn" type="ScheduleModeBitmap" min="0x0" max="0x3"/>
     </command>
-    <command source="client" code="0x03" name="ClearWeeklySchedule" optional="true">
-      <description>
-        The Clear Weekly Schedule command is used to clear the weekly schedule.
-      </description>
+    <command source="client" code="0x03" name="ClearWeeklySchedule" optional="true" disableDefaultResponse="true">
+      <description>This command is used to clear the weekly schedule. The Clear weekly schedule has no payload.</description>
       <access op="invoke" privilege="manage"/>
     </command>
     <command source="server" code="0x00" name="GetWeeklyScheduleResponse" optional="true">
-      <description>
-        The Current Weekly Schedule Command is sent from the server in response to the Get Weekly Schedule Command.
-      </description>
+      <description>This command has the same payload format as the Set Weekly Schedule. Please refer to the payload detail in </description>
       <arg name="NumberOfTransitionsForSequence" type="int8u"/>
       <arg name="DayOfWeekForSequence" type="ScheduleDayOfWeekBitmap" min="0x0" max="0xFF"/>
       <arg name="ModeForSequence" type="ScheduleModeBitmap" min="0x0" max="0x3"/>
       <arg name="Transitions" type="WeeklyScheduleTransitionStruct" array="true" length="10"/>
     </command>
+    <command source="client" code="0x06" name="SetActivePresetRequest" optional="true">
+      <description>Upon receipt, this SHALL set the thermostat</description>
+      <arg name="PresetHandle" type="octet_string" length="16"/>
+      <arg name="DelayMinutes" type="int16u" optional="true" default="0"/>
+    </command>
+    <command source="client" code="0x09" name="CommitPresetsSchedulesRequest" optional="true">
```

Comment:
Again, why the weird ordering here?  This time the spec _is_ in numeric order....